params/config.go

```diff
@@ -49,6 +49,7 @@ var (
 		KoreCompatibleBlock:      big.NewInt(119750400),
 		Kip103CompatibleBlock:    big.NewInt(119750400),
 		ShanghaiCompatibleBlock:  big.NewInt(135456000),
+		CancunCompatibleBlock:    big.NewInt(0xffffffff),
```

Comment:
As far as I know, nil is okay. What about using nil?
```
...(nil = no fork, 0 already on Cancun)
```

Reply:
Done in 0b8adebe, thanks.